go.mod

```diff
@@ -33,7 +33,7 @@ require (
 	k8s.io/apiserver v0.31.0
 	k8s.io/client-go v0.31.0
 	k8s.io/component-base v0.31.0
-	k8s.io/cri-api v0.31.0
+	k8s.io/cri-api v0.32.0-alpha.0
```

Comment:
was this intentional?

Reply:
no

Reply:
but it comes from containerd v2.0.0, so I guess it's intentional :)